SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec

```diff
@@ -0,0 +1,229 @@
+# This spec file assumes you're building on an environment where:
+# * You have access to the internet during the build
+# * You have rustup installed on your system
+# * You have both x86_64-unknown-linux-gnu and x86_64-unknown-linux-musl, or
+# * You have both aarch64-unknown-linux-gnu and aarch64-unknown-linux-musl
+#   targets installed.
+
+%define using_rustup 0
+%define using_musl_libc 0
+%define using_vendored_crates 1
+
+# Generated by update script
+%global gitver   38.0
+%global gitnum   72
+%global gitrev   8f7209bdc
+%global clh_name cloud-hypervisor
+%global clh_type cvm
+
+Name:           %{clh_name}-%{clh_type}
+Summary:        Cloud Hypervisor CVM is an open source Virtual Machine Monitor (VMM) that enables running SEV SNP enabled VMs on top of MSHV using the IGVM file format as payload.
+Version:        %{gitver}.%{gitnum}
+Release:        1000.%{gitrev}%{?dist}
+License:        ASL 2.0 or BSD-3-clause
+Group:          Applications/System
+URL:            https://github.com/cloud-hypervisor/cloud-hypervisor
+Source0:        https://github.com/microsoft/cloud-hypervisor/archive/refs/tags/msft/v%{version}.tar.gz#/%{clh_name}-%{version}.tar.gz
```

Comment:
Our sources may collide with proper cloud-hypervisor. Any chance we can still rename the tar (and vendor) file to contain clh_type?

Reply:
They should not because clh-cvm always has the patch number in its version (eg 38.0.72). Vs upstream only uses major and minor (eg 38.0). Still good to make it clear these are cvm sources

Reply:
Will also need to adapt Kata CI to rename the tars (add `-cvm` ) given by LSG

Reply:
Just double checked and this isn't as straightforward as expected. The macro `%setup` expects the untared folder to also be called *-cvm

<img width="589" alt="image" src="https://github.com/microsoft/azurelinux/assets/30801614/de1c5b35-a623-4e72-9071-29c8aa95b3d3">

And LSG doesn't renames them to *-cvm. We could rename them ourselves, but this would alter the hash sum. This makes me want to leave them as is (eg `cloud-hypervisor-38.0.72.tar.gz`), and rely on the fact that there won't be a name collision with normal clh sources, because clh-cvm will always have a patch number in its source tar